taggit_selectize/widgets.py

```diff
@@ -41,7 +41,7 @@ def render(self, name, value, attrs=None, renderer=None):
             <script type="text/javascript">
                 (function($) {
                     $(document).ready(function() {
-                        $("input[id=%(id)s]").selectize({
+                        django.jQuery("input[id=%(id)s]").selectize({
```

Comment:
Can you try including jQuery? This change is not complete because $ is referenced in line 43 as well.